doc/changelog/10-standard-library/14008-Cantor-pairing.rst

```diff
@@ -0,0 +1,9 @@
+- **Added:**
+  Cantor pairing function (and its inverse) to ``Arith``.
+  ``Cantor.to_nat : nat * nat -> nat`` and ``Cantor.of_nat : nat -> nat * nat``
+  are the respective bijections between ``nat * nat`` and ``nat``.
+  ``Cantor.cancel_of_to`` and ``Cantor.cancel_to_of`` are the canceling lemmas.
+  ``Cantor.to_nat_inj`` and ``Cantor.of_nat_inj`` are the injectivity corollaries.
+  The lemma ``Cantor.to_nat_non_decreasing x y : x + y <= to_nat (x, y)`` is useful for size recursion.
```

Comment:
Perhaps it would help the reader find your addition if you mention the file `Cantor.v`?  That it's in Arith seems less important.

My impression is that line 2 is sufficient as a summary of what's been added and that lines 3-7 might be better as comments in `Cantor.v` so that info can be easily found.  The changelog is just to bring things to people's attention.

WDYT?

Reply:
Sure, I can make it less verbose.